java/list-kml-contents/README.md

```diff
@@ -14,8 +14,9 @@ The contents of the KML file are shown in a tree. Select a node to zoom to that
 
 ## How it works
 
-1. Add the KML file to the scene as a layer.
-2. Explore the root nodes of the `KmlDataset` recursively explored to create a view model.
+1. Create a `KmlLayer` from the `KmlDataset` which represents the KML file.
+1. Add the `kmlLayer` to the `SceneView`'s operational layers.
```

Comment:
we can just refer to the kml layer is plain english on this line since we "instantiated" it above 
```suggestion
1. Add the KML layer to the `SceneView`'s operational layers.
```